### PR TITLE
footer done

### PR DESCRIPTION
**Pull request by MuhammadJibranBaig**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,42 +2,58 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+      integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr"
+      crossorigin="anonymous"
```
Comment (alihassan18) [inline public/index.html:12]: don't use font awesome please remove this link

File: src/components/footer/index.js [hunk 2]
```diff
@@ -3,20 +3,28 @@ import { Link } from 'react-router-dom';
 import { Col, Container, Row } from 'reactstrap';
 import { Logo } from 'src/assets/images';
 import styled from 'styled-components';
+const FooterWrapper = styled.div`
+  padding: 2rem;
+  padding-bottom: 0rem;
+`;
 const Logoo = styled.img`
```
Comment (alihassan18) [inline src/components/footer/index.js:10]: Chane name of logo to Logo

File: src/components/footer/index.js [hunk 3]
```diff
@@ -3,20 +3,28 @@ import { Link } from 'react-router-dom';
 import { Col, Container, Row } from 'reactstrap';
 import { Logo } from 'src/assets/images';
 import styled from 'styled-components';
+const FooterWrapper = styled.div`
```
Comment (alihassan18) [inline src/components/footer/index.js:6]: Change FooterWrapper to Wrapper

File: src/components/footer/index.js [hunk 4]
```diff
@@ -30,84 +38,165 @@ const Paragraph = styled.p`
...
-                      alt="logo-proper-six"
-                      class="img-footer"
-                    />
-                  </a>
+        <FooterWrapper>
+          <Container>
+            <Row>
+              <Col lg="6" md="12" xs="12">
+                <Col lg="12" md="12" xs="12">
+                  <UnorderList>
```
Comment (alihassan18) [inline src/components/footer/index.js:93]: Change UnorderList to List

File: src/components/footer/index.js [hunk 5]
```diff
@@ -30,84 +38,165 @@ const Paragraph = styled.p`
...
+                    <Link href="#">
+                      <Logoo
```
Comment (alihassan18) [inline src/components/footer/index.js:95]: Change Logoo to Logo

File: src/components/footer/index.js [hunk 6]
```diff
@@ -30,84 +38,165 @@ const Paragraph = styled.p`
...
+                  <FooterList>
+                    <Link href="#">Information Center</Link>
+                  </FooterList>
+                  <FooterList>
+                    <Link to="#">Newsletter</Link>
+                  </FooterList>
+                </UnorderList>
+              </Col>
+              <Col lg="2" md="4" xs="12">
+                <UnorderList>
```
Comment (alihassan18) [inline src/components/footer/index.js:138]: Use List as wrapper and ListItem as a child

File: src/views/landing/components/LandingPage.js [hunk 7]
```diff
@@ -1,6 +1,34 @@
 import React from 'react';
+import { Card, CardText, CardTitle, Col, Container } from 'reactstrap';
 const LandingPage = () => {
-  return <div>ali</div>;
+  return (
+    <Container>
+      <Col lg="6" md="6" sm="6">
+        <Card>
+          <div className="cards-body">
```
Comment (alihassan18) [inline src/views/landing/components/LandingPage.js:8]: Use styled components

File: src/views/landing/components/LandingPage.js [hunk 8]
```diff
@@ -1,6 +1,34 @@
...
+            <div className="body-content">
+              <div className="body-inner-div">
+                <span className="img-div">{/* <img src={marketCap} /> */}</span>
+                <CardTitle>Circulation Amount</CardTitle>
+              </div>
+              <div>
+                <CardText>N/A</CardText>
+              </div>
+            </div>
+            <div className="body-content">
```
Comment (alihassan18) [inline src/views/landing/components/LandingPage.js:18]: Don't use classes

File: src/components/footer/index.js (hunk 6 above)
Comment (MuhammadJibranBaig) [reply src/components/footer/index.js:138]: Wrapper is already used instead of wrapper I used List

**Pull request closed without merging**